CHANGES.rst

```diff
@@ -38,6 +38,9 @@ Release History
 
 **Changed**
 
+- Python 2 is no longer supported. The oldest supported Python version is 3.4.
```

Comment:
This will need to be fixed up when 5ce285f4deb81437b1dc90f7ccaecb0b934c65ac is in master (#1514).

Reply:
Done